bin/citrea/tests/all_tests.rs

```diff
@@ -12,5 +12,5 @@ mod test_helpers;
 
 const DEFAULT_MIN_SOFT_CONFIRMATIONS_PER_COMMITMENT: u64 = 1000;
 const DEFAULT_DEPOSIT_MEMPOOL_FETCH_LIMIT: usize = 10;
-const DEFAULT_PROOF_WAIT_DURATION: u64 = 300; // 5 minutes
+const DEFAULT_PROOF_WAIT_DURATION: u64 = 600; // 10 minutes
```

Comment:
Mostly, this looks like the fix... am i correct?

Reply:
yeah. we'll see on ci if this is enough

Reply:
for mock da 5 min was more than enough